packages/wallets/src/evm/connectors/smart-wallet/lib/paymaster.ts

```diff
@@ -75,6 +75,11 @@ class VerifyingPaymasterAPI extends PaymasterAPI {
         }`;
         headers["x-authorize-wallet"] = "true";
       }
+
+      headers["x-sdk-version"] = (globalThis as any).X_SDK_VERSION;
+      headers["x-sdk-name"] = (globalThis as any).X_SDK_NAME;
+      headers["x-sdk-platform"] = (globalThis as any).X_SDK_PLATFORM;
+      headers["x-sdk-os"] = (globalThis as any).X_SDK_OS;
```

Comment:
same here, fallback inline, otherwise its too dependent on who sets up their globals first

Reply:
We desperately need a utils package to share between all packages